external-crates/move/crates/move-cli/src/base/test.rs

```diff
@@ -170,7 +171,7 @@ pub fn run_move_unit_tests<W: Write + Send>(
                 .map(|fname| {
                     let contents = fs::read_to_string(Path::new(fname.as_str())).unwrap();
                     let fhash = FileHash::new(&contents);
-                    (fhash, (*fname, contents))
+                    (fhash, (*fname, Arc::from(contents)))
```

Comment:
I appreciate these changes, and I know Adam made them, too. Can we pull them out into a separate PR to land separately?

Reply:
I'll give it a shot :)